test/rss.cpp

```diff
@@ -154,6 +154,41 @@ TEST_CASE("W3CDTF parser behaves correctly") {
 		REQUIRE(rsspp::rss_parser::__w3cdtf_to_rfc822("2008-12-30T10:03:15-08:00") == "Tue, 30 Dec 2008 18:03:15 +0000");
 	}
 
+	// There has been a problem in the C date conversion functions when the TZ is
+	// set to different locations, and localtime is in daylight savings. One of
+	// these two next tests should be in active daylight savings
+	SECTION("W3CDTF with -08:00 timezone Pacific") {
+		setenv("TZ", "US/Pacific", 1);
+		tzset();
+		REQUIRE(rsspp::rss_parser::__w3cdtf_to_rfc822("2008-12-30T10:03:15-08:00") == "Tue, 30 Dec 2008 18:03:15 +0000");
+		unsetenv("TZ");
```

Comment:
Thanks.  I added the comments as requested, and move these tests into their own test case with test sections, so the setting and reset of the TZ variable can be done properly.